spec/fixtures/6to5/double-quotes.js

```diff
@@ -1,3 +1,3 @@
-"use 6to5";
+"use babel";
```

Comment:
This file can go now right? Since all the fixtures are in the new `babel` folder?